pkg/webhook/stats_reporter.go

```diff
@@ -39,6 +39,10 @@ func init() {
 	}
 }
 
+// JULIAN - why does this interface exist?  I guess it hides the context from code outside this
+// package.  But that doesn't seem to actually do anything.  Why dont' we just make the context
+// in the reporting functions?
```

Comment:
I agree - I don't see what it does other than provide an unused layer of abstraction.

Reply:
I actually _do_ end up using an interface to mock the reporting in `pkg/mutation/system_test.go`.  Think I should leave them in for now?

Or I could just use an interface in that one location (since it's helping me test) and remove it in the others?